_episodes/02-numpy.md

```diff
@@ -393,6 +393,9 @@ standard deviation: 4.61383319712
 > to see a list of all functions and attributes that you can use. After selecting one, you
 > can also add a question mark (e.g. `numpy.cumprod?`), and IPython will return an
 > explanation of the method! This is the same as doing `help(numpy.cumprod)`.
+> If you are using the "plain vanilla" Python interpreter, you can type `numpy.` and then 
+> press tab twice to print all the functions contained in the numpy library. Then you can 
```

Comment:
I think it is a little misleading to say "all the functions contained in the numpy library" as I believe, for example, that it does not descend to show the functions available from the `numpy.ma` module, like `numpy.ma.average`, which are also part of the NumPy library. 

@drakeasberry What do you (and @annefou @maxim-belkin ) think about removing the sentence:
```
 This is the same as doing `help(numpy.cumprod)`.
```
and modify your addition to be something like:
```
> Similarly, if you are using the "plain vanilla" Python interpreter, you can type `numpy.` and
> press tab twice for a listing of what is available. Then you can use the `help` function, in the
> manner of `help(numpy.cumprod)` , to see an explanation of the function.
```

Reply:
I like your suggestion, Lauren.
To be fair, @drakeasberry did not say "recursively" and your `numpy.ma` comment hovers over that territory.

> Similarly, if you are using the "plain vanilla" Python interpreter, you can type `numpy.` and
press the \<kbd>Tab\</kbd> key twice for a listing of what is available. You can then use the `help()` function to see an explanation of the function you're interested in, for example: `help(numpy.cumprod)`.

Reply:
I like the suggested modification by Lauren as well.

Reply:
I like @maxim-belkin's further adaptation too :). thanks @drakeasberry

Reply:
Great job! I like it too and have nothing to add! Thank you!!!